modules/safer-cluster-update-variant/main.tf

```diff
@@ -63,6 +63,10 @@ module "gke" {
   // rather than individual teams.
   network_policy = true
```

Comment:
`network_policy = true` conflicts with dataplane v2 (it force-enables network policies, but the Google Cloud provider - and probably the REST API - requires that you pass false). I suggest something like this:

```
  network_policy = var.datapath_provider == "ADVANCED_DATAPATH" ? false : true
```

Reply:
Makes sense to me, will do

Reply:
Added